policy/enforcer.go

```diff
@@ -31,6 +31,27 @@ type Enforcer struct {
 	PolicyLoader Loader
 }
 
+// type for Identity Token claims
+type claims struct {
```

Comment:
```suggestion
type checkedClaims struct {
```

I got confused reading the code the type and variable both being named claims.

Reply:
Updated the name to the recommendation